src/csharp/INTEGRATION.md

```diff
@@ -0,0 +1,357 @@
+Protocol Buffers/gRPC Integration Into .NET Build
```

Comment:
nit: the name of file  INTEGRATION.md seems a bit too general - it's not immediately obvious what's meant by that.
How about CODEGEN.md  or CODEGENERATION.md, CODEGEN-INTEGRATION.md, MSBUILD-INTEGRATION.md ....?

Reply:
Most regular developers have no idea what MSBuild is, and are hardly aware that they are in fact using it when hitting Shift+Ctrl+B for "Build" in Visual Studio or typing `dotnet build`; I'd rather avoid mentioning it in a file name. The common theme is "build" here; BUILD-INTEGRATION.md maybe then?

Or TOOLS-NUGET-PACKAGE.md?

CODEGEN etc. sounds also less appealing, looking through the user's eyes. As in "I want to compile proto files." The documentation refers to protoc as the Protocol Buffer compiler; codegen may be perceived as just a teensy option to it. I'm afraid CODEGEN would not ring the bell when seen from the user's perspective.

But overall, I think that a link from README will go a long way toward rendering the name choice not very important. Absolutely most people read README files on GitHub these days, not looking for one after unpacking a tarball--not .NET users, certainly. Just let me know what name you pefer.

From your other comment:

>  I usually call them "Legacy" rather than "Classic" btw.

I never heard them called legacy anywhere in MS' own and related development. The operational jargon is "Classic projects" vs "SDK projects" (or s/project/build/ when speaking of the process, not file format). FWIW, the "classic" is not obsolete; they in fact now support ProjectReference, just like SDK projects, with NuGet tasks integrated into MBuild, in lieu of packages.conf (which relied on a plugin shipped with VS to modify the .csproj content). Or maybe I was just spending too much time digging through MSBuld, netcore and NuGet repositories while working on our thing. :)

One point statistics is not convincing, but here's a single example I just happened to hit first: a Googler discusses an issue with an MSBuild architect in this jargon: https://github.com/Microsoft/msbuild/issues/3546. I'd go with Classic even in the example name, if I were to choose. When I first seen it renamed to HelloWorldLegacy, it fell on my ear as not exactly right. Yes, I'm biased, but it seems I'm biased in the right direction this time. :)

I could not find any named "official" names in the documentation (docs.microsoft.com), so we are drawing a blank here. .NET project, Visual Stdio project. Boring. Even my cats have names, although they also do not care...